docs/faststore/docs/managing-performance/improving-store-performance-with-preact.mdx

```diff
@@ -18,7 +18,9 @@ Make sure the `@faststore/cli` package from your project is updated to the lates
 ## Instructions
 
 1. Open your FastStore project code.
-2. Go to `faststore.config.js` file and add the `preact` flag in the `experimental` section:
+2. Go to `discovery.config.js` file and add the `preact` flag in the `experimental` section:
+
+> The `faststore.config.js` file is the legacy filename for `discovery.config.js`. If you still using the legacy name, please refer to [this release note]() to update it.
```

Comment:
@Mariana-Caetano we don't have a release note on the portal related to this issue yet, right?
We have this one in GH: https://github.com/vtex/faststore/releases/tag/v3.0.123

Reply:
@hellofanny not yet, I've created this draft release: https://docs.google.com/document/d/18yZpW9Kf2QqU3EeNxDdPCBFAjhfLv-TSp83ezl556hg/edit?tab=t.0

Feel free to comment on this, I also left some questions on there.

Reply:
@hellofanny for now, I'll add the release note from GitHub, but once we have the draft ready I'll update the doc.

Reply:
Okay!! Thank you!